server.py

```diff
@@ -78,6 +78,11 @@ def index():
     return render_template('index.html', data=data)
 
 
+@app.route('/endpoints.json', methods=['GET'])
+def endpoints():
+    return jsonify({"endpoints": [{'name': x, 'parameters': y.params} for x, y in sorted(endpoints.items())]})

```

Comment:
Does this *need* to be sorted?

Reply:
Not necessarily, but it would be nice perhaps

Reply:
alphabetical order or bean! Well if you think it's useless I can remove it in a jiffy.

Reply:
Tbf if the primary use-case is to dynamically support endpoints within bots then A-Z sorting isn't a huge priority here and just adds a few ms of overhead at most

Reply:
I agree.